integration/ng-add-standalone/package.json

```diff
@@ -10,24 +10,24 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "file:../../node_modules/@angular/animations",
+    "@angular/animations": "^15.0.0",
```

Comment:
@devversion I had to change these dependencies, because they were causing a mismatch which was breaking the build. I saw that we already had something similar in the Yarn PnP test as well.

Reply:
I'm good with this. Out of curiosity: If the root package.json would be using `15.0.0` too, why would it be different this way?

Reply:
I'm not sure, my guess is that a transitive dependency somewhere was causing it. I actually only needed to set a specific version for `@angular-devkit/build-angular` to fix the problem, but it would've looked weird if this was the only package with a version.

Reply:
Interesting. I can tell you that dependencies like `@angular/animations` will regardless end up using the workspace `@angular/animations` package (due to the integration Bazel rule), but yes- there might be something with transitive dependencies.

Also `build-angular` will end up being the one from the project `package.json`: https://github.com/angular/components/blob/main/tools/integration.bzl#L12.